scripts/pl-install.sh

```diff
@@ -1,44 +1,44 @@
 #!/bin/bash
 set -ex
 
-# If you need to rebuild this image without actually changing anything,
-# add a dot to the following line:
-# .
+export DEBIAN_FRONTEND=noninteractive
 
-dnf update -y
+apt-get update -y
+apt-get upgrade -y
+
+# Add PostgreSQL APT repository for PostgreSQL 16 (from https://www.postgresql.org/download/linux/ubuntu/)
+apt-get install -y curl ca-certificates
+install -d /usr/share/postgresql-common/pgdg
+curl -o /usr/share/postgresql-common/pgdg/apt.postgresql.org.asc --fail https://www.postgresql.org/media/keys/ACCC4CF8.asc
+echo 'deb [signed-by=/usr/share/postgresql-common/pgdg/apt.postgresql.org.asc] https://apt.postgresql.org/pub/repos/apt noble-pgdg main' > /etc/apt/sources.list.d/pgdg.list
 
 # Notes:
-# - `gcc-c++` is needed to build the native bindings in `packages/bind-mount`
-# - `libjpeg-devel` is needed by the Pillow package
-# - `procps-ng` is needed for the `pkill` executable, which is used by `zygote.py`
-# - `texlive` and `texlive-dvipng` are needed for matplotlib LaTeX labels
-dnf -y install \
+# - `g++` (via build-essential) is needed to build the native bindings in `packages/bind-mount`
+# - `libjpeg-dev` is needed by the Pillow package
+# - `procps` is needed for the `pkill` executable, which is used by `zygote.py`
+# - `texlive`, `dvipng`, and `texlive-latex-extra` are needed for matplotlib LaTeX labels
+#   (type1cm is only available in texlive-latex-extra, see https://github.com/matplotlib/matplotlib/issues/27654)
+apt-get install -y --no-install-recommends \
     bash-completion \
-    gcc \
-    gcc-c++ \
+    build-essential \
+    curl \
+    dvipng \
     git \
     graphviz \
-    graphviz-devel \
-    ImageMagick \
-    libjpeg-devel \
+    libgraphviz-dev \
+    imagemagick \
+    libjpeg-dev \
     lsof \
-    make \
     openssl \
-    postgresql16 \
-    postgresql16-server \
-    postgresql16-contrib \
-    procps-ng \
-    redis6 \
+    postgresql-16 \
+    postgresql-contrib-16 \
+    procps \
+    redis-server \
     tar \
     texlive \
-    texlive-dvipng \
-    texlive-type1cm \
+    texlive-latex-extra \
```

Comment:
I'm curious how you selected these packages. `texlive-base` exists as an alternative to `texlive` and is smaller; `texlive-latex-extra` is ~775MB installed, and seems to be composed of a lot of specific packages that we could pull in if needed.

EDIT: I see a note that a needed style is only available via `textlive-latex-extra`. Is there really _no_ other way to install it? Pulling in hundreds of megabytes of stuff for that seems unreasonable.

Reply:
I could not find another `apt` package with that info, you are welcome to look yourself.